docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -393,6 +393,10 @@ children:
   url: FAQ.html
   output: 'web, pdf'
 
+- title: 'API docs'
+  url: apidocs.index.html
```

Comment:
Is there any particular reason for placing all apidocs files in the same directory and prefixing their file names?

Personally, I'd prefer `apidocs/index.html` for the index file, and `apidocs/{package}.{class}.{member}.html` for documentation files.

Reply:
Yes. I initially had `apidocs/index.html` but it messed up the sidebar links. For apidocs pages, all links become `apidocs/*`.

Reply:
Fair enough.